src/test/java/com/hocs/test/pages/create_case/CreateCase.java

```diff
@@ -38,10 +52,20 @@
     @FindBy(id = "")
     private WebElementFacade addDocumentsYesRadioButton;
 
+
+    // Basic Methods
+
     public void assertNoOptionsAvailable() {
         assertThat(allRadioButtons.getText(),is("No options available"));
     }
 
+    public void capturedCaseReferenceTest() {
```

Comment:
Is this method still valid? You could at the debug to the method that sets the session variable